src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -63,11 +86,123 @@ const NavMenu = (): ReactElement => {
             </NextLink>
           </Grid>
           <Grid item className={classes.navListItem}>
-            <HandoffLink path="/reports">
-              <MenuItem component="a">
-                <ListItemText primary={t('Reports')} />
-              </MenuItem>
-            </HandoffLink>
+            <MenuItem
+              ref={anchorRef}
+              aria-controls={reportsMenuOpen ? 'menu-list-grow' : undefined}
+              aria-haspopup="true"
+              onClick={handleReportsMenuToggle}
+            >
+              <ListItemText primary={t('Reports')} />
+            </MenuItem>
+            <Popper
+              open={reportsMenuOpen}
+              anchorEl={anchorRef.current}
+              role={undefined}
+              transition
+              disablePortal
+            >
+              {({ TransitionProps, placement }) => (
+                <Grow
+                  {...TransitionProps}
+                  style={{
+                    transformOrigin:
+                      placement === 'bottom' ? 'center top' : 'center bottom',
+                  }}
+                >
+                  <Paper>
+                    <ClickAwayListener onClickAway={handleReportsMenuClose}>
+                      <MenuList
+                        autoFocusItem={reportsMenuOpen}
+                        id="menu-list-grow"
+                      >
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/donations"
+                          as={`/accountLists/${state.accountListId}/reports/donations`}
```

Comment:
Is `as` needed to make this work? Seems like it might be an older thing https://nextjs.org/docs/api-reference/next/link.

Reply:
I can give it a try... I noticed the other links on this page used `as`, I think it has to do with inserting the `accountListId` into the URL?